### DISCUSSION TITLE
SMTP error

### DISCUSSION THREAD

**Discussion by smarkitan** [Peppermint-Lab/peppermint#345] [Q&A]
Hi. I'm trying to setup the outbound email. When I did update & testing I received in console this error: 
"/api/v1/config/email/verify:1        Failed to load resource: the server responded with a status of 500 (Internal Server Error)"
Any Help please, as I haven't found through all the email discussions. Thanks a lot!

<img width="697" alt="smpt setup" src="https://github.com/Peppermint-Lab/peppermint/assets/97811066/1802c96d-0b93-40f0-976e-2e56283decc9">

**Comment by potts99** [comment]
If this is google you need an app password, there is now the option to auth via oauth 👍
